modules/apps/commerce/commerce-frontend-js/webpack.config.dev.js

```diff
@@ -69,7 +69,26 @@ module.exports = {
 				use: [
 					{loader: 'style-loader'},
 					{loader: 'css-loader'},
-					{loader: 'sass-loader'},
+					{
+						loader: 'sass-loader',
+						options: {
+							sassOptions: {
+								importer: (url, _, done) => {
+									if (url.includes('atlas-variables')) {
+										done({
+											file: path.resolve(
+												__dirname,
+												'../../../node_modules/@clayui/css/src/scss/atlas-variables.scss'
+											),
+										});
+									}
+									else {
+										done({file: url});
+									}
+								},
+							},
+						},
+					},
```

Comment:
It's pretty "funky" that you need this, but seeing as it is local to this one module I think that's ok. If this kind of thing crops in more than one place though we should take a deeper look.

Reply:
Yes. I tried other ways to make our static env working (that import broke the bundling) but the only working one I've found is configuring aliases via `sassOptions`. Maybe there's a different / better solution and I missed it (please let me know if you investigate more)